environment-condalock.yml

```diff
@@ -3,6 +3,6 @@ channels:
   - conda-forge
 dependencies:
   - python=3.10
```

Comment:
Bump to python=3.11 directly here or in #461?

Reply:
lets do it in the other one! thanks @weiji14 !